### PR TITLE
Ana/fix emoney gasprices

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

I had to rename the column name on DB

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: api/lib/reducers/emoneyV0-reducers.js [hunk 1]
```diff
@@ -13,9 +13,11 @@ async function totalBackedValueReducer(
   const lunieCoin = reducers.coinReducer(totalBackedValue, coinLookup)
 
   // The total net EUR value of the token's total supply
-  const fiatValue = BigNumber(lunieCoin.amount)
-    .times(exchangeRates[lunieCoin.denom][aggregatingCurrency])
-    .toNumber()
+  const fiatValue = exchangeRates
```
Comment (Bitcoinera) [inline api/lib/reducers/emoneyV0-reducers.js:16]: this is useful if we are developing and don't have the emoney env variables in our `.env`

File: api/lib/reducers/emoneyV0-reducers.js [hunk 2]
```diff
@@ -68,7 +70,9 @@ async function expectedRewardsPerToken(
   // How many NGM tokens can we buy with the total gain in EUR we make in a year's time?
   // 0.50€ is the price the NGM tokens will be first sold. Therefore, this is the official value
   // until they reach an exchange
-  const pricePerNGM = exchangeRates[stakingToken][aggregatingCurrency]
+  const pricePerNGM = exchangeRates
```
Comment (Bitcoinera) [inline api/lib/reducers/emoneyV0-reducers.js:73]: same here

File: api/changes/ana_fix-emoney-gasprices [hunk 3]
```diff
@@ -0,0 +1 @@
+[Fixed] [#5182](https://github.com/cosmos/lunie/pull/5182) Fix e-Money gasprices by changing column name from gasPrice to price @Bitcoinera
```
Comment (faboweb) [inline api/changes/ana_fix-emoney-gasprices:1]: why was this necessary?

Comment (Bitcoinera) [reply api/changes/ana_fix-emoney-gasprices:1]: it wouldn't even open up the ActionModal. Gives the following error:

image

**Review by faboweb — APPROVED** [review]
